p2p/src/utils/oneshot_nofail.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Simple wrapper for the `tokio::sync::oneshot` channel
+//! that does not return an error if the receiver is disconnected.
```

Comment:
Please add here why this exists and what problem it solves (pretty much what you wrote in the PR)

Reply:
Sure, updated, thanks.